todo_backend/todo_project/settings.py

```diff
@@ -11,9 +11,20 @@
 """
 
 from pathlib import Path
-
 import os 
 
+from rest_framework import authentication 
+
+# custom authentication class to bypass csrf checks
+# The csrf check causes csrf missing errors when the react frontend was running
+# in the same browser session as the django backend, and a user was
+# logged into the django backend (for django admin)
+class CsrfExemptSessionAuthentication(authentication.SessionAuthentication):
+
+    def enforce_csrf(self, request):
+        return  # To not perform the csrf check previously happening
+
+
```

Comment:
Is this redundant with `auth.py`?  Do we need to have this in both places?

Reply:
thanks!  removed.